sonar-php-plugin/src/main/java/org/sonar/plugins/php/PhpPlugin.java

```diff
@@ -38,6 +39,8 @@
   public static final String GENERAL_SUBCATEGORY = "General";
   public static final String PHPUNIT_SUBCATEGORY = "PHPUnit";
 
+  public static final Version COVERAGE_USE_EXECUTABLE_LINES = Version.create(6, 2);
```

Comment:
@henryju I think it's better rename to smth like `SQ_VERSION_6_2`. We will reuse it for coverage type stuff as well, so IMO this description (that coverage is using executable lines) is not highly required.